OutOfSchool/OutOfSchool.BusinessLogic/Util/MappingProfile.cs

```diff
@@ -489,6 +517,20 @@ public MappingProfile()
             .ForMember(dest => dest.EmailConfirmed, opt => opt.MapFrom(src => src.User.EmailConfirmed))
             .ForMember(dest => dest.UserName, opt => opt.MapFrom(src => src.User.UserName));
 
+        CreateMap<Parent, BaseUpdateUserDto>()
```

Comment:
The `Update` DTO is going into the system. It should not be returned anywhere. So this mapping should not exist. So change the logic using this mapping to use `ShortUserDto` instead